docs/source/topics/build-examples.rst

```diff
@@ -26,24 +26,53 @@ basic usage scenarios of |short_name| with SYCL*. Go to
 
 .. note::
 
-   All content starting with a # below is considered a comment and
+   All content below that starts with ``#`` is considered a comment and
    should not be run with the code.
 
 1. Set up the |short_name| environment:
 
-  **Linux\*:**
+  .. tabs::
 
-    .. substitution-prompt:: bash
+    .. group-tab:: Linux\*
 
-      # Run script to setup CPATH, LIBRARY_PATH and LD_LIBRARY_PATH for |short_name|
-      source ./env/vars.sh
+      On Linux, there are two possible ways to set up |short_name| environment:
+      via ``vars.sh`` script or via ``modulefiles``.
 
-  **Windows\*:**
+      * Setting up |short_name| environment via ``vars.sh`` script
 
-    .. substitution-prompt:: bash
+        Run ``vars.sh`` script to set up ``CPATH``, ``LIBRARY_PATH``, and ``LD_LIBRARY_PATH`` for |short_name|:
 
-      # Run script to setup PATH, LIB and INCLUDE for |short_name|
-      /env/vars.bat
+          .. prompt:: bash
+
+            source ./env/vars.sh
+
+      * Setting up |short_name| environment via ``modulefiles``
+
+        1. Initialize ``modules``:
+
+          .. prompt:: bash
+
+            source $MODULESHOME/init/bash
```

Comment:
I would replace this by general comment that users should refer to original documentation for modulefiles configuration process.  And start with next item as first one

Reply:
We can add a reference to original documentation. But if this is the first step the user should do, I wouldn't remove this step.

Reply:
added a reference